tests/__snapshots__/inductor_left.snap.svg

```diff
@@ -0,0 +1,17 @@
16 L0.21995136999999979,0.013649213595881957 L0.22244090999999977,0.017980893952924293 L0.22493044999999978,0.022192955076609885 L0.2274199899999998,0.026274777036001493 L0.2299095299999998,0.03021654286791864 L0.23239906999999982,0.03400929910178013 L0.23488860999999983,0.03764501096072753 L0.23737814999999984,0.041116611898696254 L0.23986768999999986,0.044418047168022 L0.24235722999999987,0.047544311148973464 L0.24484676999999988,0.050491478211062556 L0.2473363099999999,0.053256726915861696 L0.2498258499999999,0.05583835741211006 L0.2523153899999999,0.05823580191586171 L0.25480492999999993,0.06044962821106255 L0.25729446999999994,0.06248153614897346 L0.25978400999999995,0.064334347168022 L0.26227354999999997,0.06601198689869625 L0.26476309,0.06751946096072754 L0.26725263,0.06886282410178013 L0.26974217,0.07004914286791863 L0.27223171,0.07108645203600147 L0.27472125000000003,0.07198370507660988 L0.27721079000000004,0.0727507189529243 L0.27970033000000005,0.07339811359588196 L0.28218987000000006,0.07393724642877121 L0.2846794100000001,0.07438014234494014 L0.2871689500000001,0.07473941957033049 L0.2896584900000001,0.07502821186791862 L0.2921480300000001,0.07526008756369948 L0.2946375700000001,0.07544896589344473 L0.29712711000000014,0.0756090311859859 L0.29961665000000015,0.07575464541211004 L0.3021061875000001,0.0756090311859859 L0.30459572500000015,0.07544896589344473 L0.3070852625000001,0.07526008756369947 L0.30957480000000015,0.07502821186791862 L0.31206433750000007,0.07473941957033048 L0.31455387500000004,0.07438014234494014 L0.31704341250000007,0.0739372464287712 L0.3195329500000001,0.07339811359588194 L0.32202248750000007,0.07275071895292427 L0.32451202500000004,0.07198370507660987 L0.3270015625,0.07108645203600146 L0.3294911,0.07004914286791862 L0.33198063750000006,0.06886282410178012 L0.334470175,0.06751946096072753 L0.3369597125,0.06601198689869624 L0.33944925,0.06433434716802199 L0.34193878749999995,0.06248153614897345 L0.344428325,0.06044962821106254 L0.3469178624999999,0.058235801915861696 L0.3494073999999999,0.055838357412110046 L0.35189693749999995,0.05325672691586168 L0.35438647499999987,0.050491478211062535 L0.3568760124999999,0.04754431114897346 L0.35936554999999987,0.04441804716802199 L0.36185508749999984,0.04111661189869624 L0.36434462499999987,0.03764501096072752 L0.3668341624999999,0.03400929910178012 L0.3693236999999998,0.03021654286791863 L0.37181323749999984,0.026274777036001475 L0.3743027749999998,0.02219295507660987 L0.3767923124999998,0.01798089395292428 L0.3792818499999998,0.013649213595881938 L0.3817713874999997,0.009209271428771202 L0.38426092499999975,0.004673092344940144 L0.3867504624999998,0.000053294570330487774 L0.38923999999999975,-0.004636988132081376 L0.3917295374999998,-0.009384187436300546 L0.3942190749999997,-0.014174384106555245 L0.3967086124999997,-0.01899339381401409 L0.3991981499999997,-0.02382685458788996" fill="none" stroke="black" stroke-width="0.02" stroke-linecap="round" stroke-linejoin="round" />
```

Comment:
So i would tweak this to flip it over the x axis (so that the bumps are upward, but still flip the ports)

Reply:
<img width="831" alt="Screenshot 2024-12-07 at 7 34 24 AM" src="https://github.com/user-attachments/assets/9ce1ad92-5cfc-49a8-9f7e-527b7135b132">

Reply:
yea that looks good!